api/CreateUserScript.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+const User = require('./models/User.js');
+const inquirer = require('inquirer');
+const mongoose = require('mongoose');
+mongoose.Promise = require('bluebird');
+
+console.log('Welcome to the SCE User Creation CLI!');
+inquirer
+  .prompt([
+    {
+      type: 'input',
+      name: 'email',
+      message: 'What\'s your email:'
+    },
+    {
+      type: 'input',
+      name: 'firstName',
+      message: 'Please enter a first name for your account (default: "Dummy"):'
+    },
+    {
+      type: 'input',
+      name: 'lastName',
+      message: 'Please enter a last name for your account (defualt: "Account"):'
+    },
+    {
+      type: 'input',
+      name: 'password',
+      message: 'Please enter a password (default: "sce"):'
+    },
+    {
+      type: 'list',
+      name: 'accountLevel',
+      message: 'Choose an account level from the list below: ',
+      choices: [
+        {
+          name: 'Admin',
+          value: '3'
+        },
+        {
+          name: 'Officer',
+          value: '2'
+        },
+        {
+          name: 'Member',
+          value: '1'
+        },
+        {
+          name: 'Non-Member',
+          value: '0'
+        },
+        {
+          name: 'Pending',
+          value: '-1'
+        },
+        {
+          name: 'Banned',
+          value: '-2'
+        }
+      ],
+      filter: function(val) {
+        console.log(val);
+        return val;
+      }
+    }
+  ])
+  .then(async (answers) => {
+    const newUser = new User({
+      password: answers.password || 'sce',
+      firstName: answers.firstName || 'Dummy',
+      lastName: answers.lastName || 'Account',
+      email: answers.email,
+      emailVerified: true,
+      accessLevel: answers.accountLevel
+    });
+    this.mongoose = mongoose;
+    this.mongoose
+      .connect('mongodb://localhost/sce_core', {
+        promiseLibrary: require('bluebird'),
+        useNewUrlParser: true,
+        useUnifiedTopology: true,
+        useCreateIndex: true
+      })
+      .then(() => {
+        console.log('MongoDB Connection Successful');
+      })
+      .catch(error => {
+        console.error(error);
```

Comment:
Instead of printing the error, print this to the user like below:

```
We couldn't connect to MongoDB. Are you sure the backend server is running?
```